about.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header class="about-header">
+    <nav class="social-media-nav">
+      <ul class="social-media">
+        <li class="social-list">
+          <a href="#"><img src="images/facebook (1).png" alt="facebook logo"></a>
+        </li>
+        <li class="social-list"><a href="#"><img src="images/twitter.png" alt="twitter logo"></a>
+        </li>
+        <li class="social-list">Spanish</li>
+        <li class="social-list">My Page</li>
+        <li class="social-list">logout</li>
+      </ul>
+    </nav>
+    <nav class="mobile-menu">
+      <div>
+        <a href="#"><img src="images/mobile_menu.png" alt="mobile menu" width="20" height="20"></a>
+      </div>
+    </nav>
+    <nav class="desktop-menu">
+      <a href="index.html"><img src="images/logo-en.svg" alt="logo culture days" width="100" height="64"></a>
+      <ul class="main-menu">
+        <li class="menu-list"><a href="about.html">About</a></li>
+        <li class="menu-list"><a href="index.html#program">Program</a></li>
+        <li class="menu-list"><a href="index.html#join">Join</a></li>
+        <li class="menu-list"><a href="index.html#sponsor">Sponsor</a></li>
+        <li class="menu-list"><a href="index.html#news">News</a></li>
+        <li class="campaign-menu"><a href="index.html">Campaign</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main class="cultural-days-wrap">
+    <section class="about">
+      <h1 class="about-tittle">ABOUT CULTURE DAYS</h1>
+      <div class="bar-icon"></div>
+      <div class="about-wrap">
+        <p class="invite-msg about-msg">Culture Days is a national celebration of arts and culture. At the end of each September, millions of people attend thousands of free participatory arts and culture events across the country both in-person and online. Culture Days programs invite the public to get hands-on and behind-the-scenes to highlight the importance of arts and culture in our communities.</p>
+      </div>
+      <p class="invite-msg contact-msg">Please contact us per Email for any questions about Culture Days 2022!</p>
+      <a class="about-email" href="#">info@culturedays.ca</a>
+    </section>
```

Comment:
- [ ] Kindly make sure the above section of the `About` page sticks as much as possible to the template design such as **background color,...**. You can check other details to match the expected design.
### Template Design
![Screenshot from 2022-09-22 20-05-22](https://user-images.githubusercontent.com/83042742/191819756-401892b7-d256-4e1e-9ada-acf3865bb360.png)

### Your Design
![Screenshot from 2022-09-22 20-06-16](https://user-images.githubusercontent.com/83042742/191819940-ce426174-d139-45af-9e86-ef6325596fa8.png)